brew/src/main/java/nl/tudelft/serg/evosql/brew/generator/junit/JUnitGenerator.java

```diff
@@ -126,17 +126,22 @@ private MethodSpec generateRunSQL() {
                             "$T connection = $T.getConnection($L, $L, $L)",
                             Connection.class, DriverManager.class,
                             NAME_DB_JDBC_URL, NAME_DB_USER, NAME_DB_PASSWORD)
-                    .addStatement("$T statement = connection.createStatement()", Statement.class)
+                    .addStatement("$T statement = connection.createStatement(ResultSet.TYPE_SCROLL_INSENSITIVE, " +
+                            "ResultSet.CONCUR_READ_ONLY)", Statement.class)
                     .beginControlFlow("if (isUpdate == true)")
                     .addStatement("statement.executeUpdate(sql)")
-                    .addStatement("return true")
+                    .addStatement("return 0")
```

Comment:
Suggestion: simply return `statement.executeUpdate` from the line above. This returns the number of matched rows. It's not really necessary but maybe it's cool to have this feature.